.github/workflows/mobile-ci.yml

```diff
@@ -159,7 +159,7 @@ jobs:
         run: yarn test
```

Comment:
@remicolin @aaronmgdr made a small tweak to the workflows to bump the mac runner. we can't bump the mobile ci test runner yet because there is a race condition in the build step [[workflow error](https://github.com/selfxyz/self/actions/runs/17308250354/job/49146191379?pr=981)]

@seshanthS has a fix in the works for the race condition. we should bump the runner after the fix gets merged